tests/main/i18n/task.yaml

```diff
@@ -26,3 +26,9 @@ execute: |
         fi
     done
     not cat bad
+
+    echo "Lint translated commands"
+    # it's fine to use ls here since the filenames are very simple
+    # shellcheck disable=SC2011
+    ls /usr/share/locale/ | xargs -Iloc apt install language-pack-loc-base -y
+    < /usr/share/i18n/SUPPORTED xargs -d '\n' -Iloc env SNAPD_DEBUG=1 LANG=loc snap changes everything 2>&1 > /dev/null | NOMATCH panic:
```

Comment:
probably using `cat /usr/share/i18n/SUPPORTED | .. ` and disabling shellcheck warning would make it a bit simpler to follow